packages/components/package.json

```diff
@@ -84,6 +87,7 @@
     "typescript-transform-paths": "^3.4.6"
   },
   "peerDependencies": {
+    "@cultureamp/i18n-react-intl": "^1.1.8",
```

Comment:
Ah probably can't be a peer, just bundle it in for now

Reply:
nw